pkgs/development/python-modules/facedancer/default.nix

```
@@ -12,15 +12,15 @@
 
 buildPythonPackage rec {
   pname = "facedancer";
-  version = "3.0.6";
+  version = "3.1.0";
   pyproject = true;
   disabled = pythonOlder "3.8";
```

Based on changelog, this should be changed to:
```suggestion
  disabled = pythonOlder "3.10";
```